library/components/tableComponents/TSelectColumns.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <button
+    class="
+      rounded
+      cursor-pointer
+      border-1 border-[#C3C2CB]
+      text-[#555463]
+      bg-white
+      w-max
+      p-1
+    "
+    style="height: fit-content"
+    :disabled="is_loading"
+    @click="download"
+  >
```

Comment:
Is it possible to use `BaseButton`?

Reply:
Absolutely, will update :D

Reply:
oh, interesting. I see that this button has the csv download information in it, and I don't see any checkbox elements anymore? Was there something that went wrong during the renaming?

Reply:
I copied and pasted the csv update into the wrong file, fixed..... 😖